openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -369,7 +369,7 @@ def __init__(self, project_name, parent):
         cancel_btn = QtWidgets.QPushButton("Cancel", self)
         cancel_btn.setToolTip("Cancel deletion of the project")
         confirm_btn = QtWidgets.QPushButton("Permanently Delete Project", self)
-        confirm_btn.setObjectName("DeleteButton")
+        confirm_btn.setObjectName("WarningButton")
```

Comment:
Is it correct that GitHub is showing these style changes for Project Manager in this PR? Or has there been a merge conflict?

Reply:
I've changed the object name because it's using same colors and has same purpose.